src/pages/SearchPage.js

```diff
@@ -160,10 +135,20 @@ class SearchPage extends Component {
     }
 
     render() {
-        const sections = this.getSections();
+        const {
+            recentReports,
+            personalDetails,
+            userToInvite,
+        } = OptionsListUtils.getSearchOptions(
```

Comment:
Ah, can we move this back to where it was? I think we should focus on just one change at a time. I only wanted to mention a possible improvement, but it feels too distracting to include in this PR.

Reply:
sure!

Reply:
Reverted